backend/storage/storage.go

```diff
@@ -380,21 +382,37 @@ func (f *FilesystemClient) writeFSBytes(ctx context.Context, fp string, data io.
 }
 
 func (f *FilesystemClient) SetupHTTPSListener(r chi.Router) {
-	r.Get("/direct/assets/{project-id}/{hash-val}", func(w http.ResponseWriter, r *http.Request) {
+	r.Use(cors.New(cors.Options{
+		AllowCredentials: true,
+		AllowedOrigins:   []string{"localhost:3000"},
```

Comment:
is hardcoding this an issue or is `FilesystemClient` only for local builds?

Reply:
good catch! this would be only used in localhost but we already use a env var provided value here (`f.origin`) so we might as well use that here too. updated!

Reply:
![image](https://github.com/highlight/highlight/assets/1351531/79d69fe8-8875-482f-87e0-900cb3f84d39)
after 63c6887ab2a3dd370c9b3a9c603ec6bf1cb0248f